### PR TITLE
Fixed issue #26 and issue #33.

### PR DESCRIPTION
**Pull request by DrMerfy**
tab_change

### CODE REVIEW

File: genume/view/event_panels.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    # Convert the left range into a 0-1 range (float)
+    valueScaled = float(value - leftMin) / float(leftSpan)
+
+    # Convert the 0-1 range into a value in the right range.
+    return rightMin + (valueScaled * rightSpan)
+
+"""
+A VBox of fixed size with added event listeners.
+"""
```
Comment (MaanooAk) [inline genume/view/event_panels.py]: Move docstrings after, not before

**Comment by drazioti** [comment]
Better (stylistic) result than the previous gui. I think the genume tab is in the wrong place and in my opinion is useless. The role of this tab is to appear the authors? What is the point of that? Isen't enough to provide credits in the github?

**Comment by MaanooAk** [comment]
@drazioti The genume folder is at the moment here for testing/sample scripts, it has a list value (how to parse a list based on the protocol, git author was the first thing it came up) and a value that changes (so the refresh functionality could be tested, the date is always changing)

**tltr:** The genume folder is for testing

**Comment by drazioti** [comment]
I believe this. But it is good to have this before I merge your pull request with my (local) master branch. If I understand write, a possible merging of this pull request will destroy all the current changes on the CSD-FOSS-Team:master. So you must first update your local branch with the remote branch (CSD-FOSS-Team:master) and check again your changes.

**Comment by drazioti** [comment]
ναι, καλό είναι. Βεβαια τα κουμπακια επάνω αριστερα (refresh) και πανω δεξια (που δεν εχω ιδεα τι είναι) δεν εμφανιζόνται σωστα. Επίσης, πατώντας τα κουμπακια πανω δεξια, ο υπολογιστης μου εκανε log off!

**Comment by DrMerfy** [comment]
The options button (upper right) can be removed, unless we plan to add some functionality to it. I am also looking on changing the position of the refresh button.

**Comment by drazioti** [comment]
The problem to me, is not the position, but I got the warning message
_Error loading theme icon 'view-refresh' for stock: Icon 'view-refresh' not present in theme_

**Comment by DrMerfy** [comment]
This may be caused by a missing icon on the theme you are using. The plan is to implement the refresh inside the GUI, so it won't be dependent on the theme you are using.

**Comment by drazioti** [comment]
that would be great, since it also solves my previous problem.

**Pull request merged**